python/lsst/daf/butler/tests/cliLogTestBase.py

```diff
@@ -168,6 +171,25 @@ def test_helpLogReset(self):
 
         self.runTest(partial(self.runner.invoke, butlerCli, ["command-log-settings-test", "--help"]))
 
+    def testLongLog(self):
+        """Verify the timestamp is in the log messages when the --long-log
+        flag is set."""
+        with self.runner.isolated_filesystem():
+            # The click test does not capture logging emitted from lsst.log
```

Comment:
Butler tests should never be issuing lsst.log log messages so lsstLog is only ever true if someone runs these tests after they've setup some other LSST package. Neither Jenkins nor GitHub Actions will ever have lsst.log available. I'm now a bit worried about these tests -- no code inside butler should ever be issuing an lsst.log log message so how do we ever see the change? Does python logging change in click subcommands based on lsst.log existing?

Reply:
This test is run by `test_cliClog.py` in `daf_butler` as well as in `test_cliLog.py` in `obs_base`, so both variants are tested.

Reply:
Can you help my future self by adding a comment here that says that this check is specifically because subclasses can enable lsst.log even if daf_butler does not use it?